core/runtime/jsruntime.rs

```diff
@@ -1219,9 +1222,8 @@ impl JsRuntime {
 
   /// Returns the runtime's op names, ordered by OpId.
   pub fn op_names(&self) -> Vec<&'static str> {
-    let main_realm = self.inner.main_realm.clone();
-    let state_rc = main_realm.0.state();
-    state_rc.op_ctxs.iter().map(|o| o.decl.name).collect()
+    let state = &self.inner.main_realm.0.context_state;
```

Comment:
This was unintentionally causing a leak of the realm. `ManuallyDrop::clone` is bad. I wish we could forbid it somehow.

Reply:
I think you could use `clippy.toml` and `disallowed-methods` there?

Reply:
Oh, let me try that out...